17thJan/armstrong.rb

```diff
@@ -0,0 +1,22 @@
+puts "Enter any number:"
+
+n=gets.chomp.to_i
+
+num=n
+arm=0
+
+for i in 1..n do
```

Comment:
Why we have this for loop here ?
This will iterate more times and takes more execution time..
Please reduce execution time...